test/Microsoft.AspNetCore.SystemWebAdapters.CoreServices.Tests/Microsoft.AspNetCore.SystemWebAdapters.CoreServices.Tests.csproj

```diff
@@ -7,6 +7,7 @@
     <PackageReference Include="AutoFixture" Version="4.15.0" />
     <PackageReference Include="Moq" Version="4.16.1" />
     <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="6.0.13" />
+    <PackageReference Include="System.Collections.Immutable" Version="8.0.0" />
```

Comment:
Unrelated to your PR, but we should think about setting up dependabot or some other way to automatically bump these dependencies in the future.